karma.conf.js

```diff
@@ -12,20 +10,28 @@ module.exports = function (config) {
     // Only test IE latest.
     config.browsers.push('internet_explorer_11');
   }
+  var polyOptions = process.argv.filter((val) => {
```

Comment:
I get it, it would just turn several LoC into a single one and normalise how the argument is passed which makes passing args a little bit less brittle.